### DISCUSSION TITLE
Filter music folder?

### DISCUSSION THREAD

**Discussion by coolthxmcdoom** [deckerst/aves#1362] [Q&A] [answered]
Hello, I have a bunch of albums in Aves of all my music cover art etc. Can I filter all of these by excluding the music directory somehow? 

Searching around I found a few other posts suggesting that there should be an option in 'Settings > Privacy > Hidden items > Hidden paths' but I genuinely have no options for such in the hidden items settings page so unsure if I'm looking in the wrong place or this option was moved or what. Any help here will be appreciated, thank you.

**Comment by deckerst** [comment, marked as answer]
Yes, it was removed from there, but the feature still exists. To do what you want, go to the Explorer page and navigate to the root of your music collection, them in the top right menu select "hide".

**Reply by coolthxmcdoom** [reply]
I see, that did what I was after. Thank you so much.
